src/logic/dataLeecher.tsx

```diff
@@ -0,0 +1,6 @@
+import ServerLogic from './DataUploader';
+
+export default function dataLeecher(data: any){
```

Comment:
so this function will allow functions to subscribe?

Reply:
yes, wherever you want to forward the reading you can forward from here, as the name implies this function get every data which is passed between source to graph.